client/shipping/woocommerce-services-item.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { Button, ExternalLink } from '@wordpress/components';
+import { Pill } from '@woocommerce/components';
+import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+import { getAdminLink } from '@woocommerce/wc-admin-settings';
+
+const WooCommerceServicesItem: React.FC< {
+	pluginsBeingSetup: Array< string >;
+	onSetupClick: ( slugs: string[] ) => PromiseLike< void >;
+} > = ( { onSetupClick, pluginsBeingSetup } ) => {
+	const { createSuccessNotice } = useDispatch( 'core/notices' );
+
+	const isSiteConnectedToJetpack = useSelect( ( select ) =>
+		select( PLUGINS_STORE_NAME ).isJetpackConnected()
+	);
+
+	const handleSetupClick = () => {
+		onSetupClick( [ 'woocommerce-services' ] ).then( () => {
+			const actions = [];
+			if ( ! isSiteConnectedToJetpack ) {
+				actions.push( {
+					url: getAdminLink( 'plugins.php' ),
+					label: __(
+						'Finish the setup by connecting your store to Jetpack.',
+						'woocommerce-admin'
+					),
+				} );
+			}
+
+			createSuccessNotice(
+				__(
+					'🎉 WooCommerce Shipping is installed!',
+					'woocommerce-admin'
+				),
+				{
+					actions,
+				}
+			);
+		} );
+	};
+
+	return (
+		<div className="woocommerce-list__item-inner">
+			<div className="woocommerce-list__item-before">
+				<img
+					src="https://woocommerce.com/wp-content/plugins/wccom-plugins/marketplace-suggestions/icons/woocommerce.svg"
```

Comment:
IIRC, we cannot directly link to external images from this repo so we may need to add this SVG to this repo.  cc @mattsherman to make sure I'm remembering this correctly.

Reply:
Thanks for the note! I'm going to ask marketing/UX if they have a different icon they'd like to use and update this once it's confirmed 👍

Reply:
New image asset replaced in 5422fed8e
![Screen Shot 2021-08-05 at 2 16 18 PM](https://user-images.githubusercontent.com/273592/128408451-3b549997-2bd1-451d-b147-85ec1dbc9c82.png)